main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -2271,7 +2271,7 @@ public void run() {
                     cache.setCoords(wpt.getCoords());
                     cache.setUserModifiedCoords(false);
                     cache.deleteWaypointForce(wpt);
-                    DataStore.saveChangedCache(cache);
```

Comment:
I don't have an IDE here, therefore I can't check. Might it be that saveChangedCache and saveUserModifiedCoords are always called together? If so, I would move the new call into saveChangedCache instead of having a separate call afterwards.

Reply:
I'll check

Reply:
There is no `saveChangedCache` anymore, or do I overlook something?

Reply:
Oh, my fault. Seems I did not really see the red color before. :)